common/flatpak-dir.c

```diff
@@ -7932,10 +7934,22 @@ flatpak_dir_check_parental_controls (FlatpakDir    *self,
   authorized = polkit_authorization_result_get_is_authorized (result);
 
   if (!authorized)
-    return flatpak_fail_error (error, FLATPAK_ERROR_PERMISSION_DENIED,
-                               /* Translators: The placeholder is for an app ref. */
-                               _("Installing %s is not allowed by the policy set by your administrator"),
-                               ref);
+    {
+      /* Send a metrics event so we have an idea if there are any places which
+       * don’t block app installing from the UI. If everything is implemented
+       * correctly, this event should never be submitted.
+       *
+       * See: https://phabricator.endlessm.com/T28741 */
+#define FLATPAK_PARENTAL_CONTROLS_INSTALL_EVENT "9d03daad-f1ed-41a8-bc5a-6b532c075832"
```

Comment:
This needs to be added to the wiki, and similarly for the `run` event.

Reply:
Both done. https://phabricator.endlessm.com/phriction/diff/545/?l=41&r=42